src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -241,6 +244,7 @@ case class Function(
                      override val pres: Vector[Assertion],
                      override val posts: Vector[Assertion],
                      override val terminationMeasures: Vector[TerminationMeasure],
+                     val privateSpec: Option[PrivateSpec],
```

Comment:
`override val`?

Reply:
I am not too sure, but `override val privateSpec: Option[PrivateSpec]` would mean I need to declare it inside:
```
sealed trait FunctionLikeMemberOrLit extends Node {
  def args: Vector[Parameter.In]
  def results: Vector[Parameter.Out]
  def pres: Vector[Assertion]
  def posts: Vector[Assertion]
  def terminationMeasures: Vector[TerminationMeasure]
}
```
And `FunctionLit extends FunctionLitLike` which extends `FunctionLikeMemberOrLit`. I am assuming that `FunctionLit` is a function declared within another function, and there seems to be no need for a `privateSpec` inside there. That is why I did not create a definition inside the `FunctionLikeMemberOrLit `

Reply:
Ok, makes sense.